frontend/src/App.js

```diff
@@ -0,0 +1,457 @@
+                        }
+                      );
+                    }
+                  }}
+                >
+                  -
+                </button>
+                {item.qty}
+
+                <button
+                  onClick={() => {
+                    cart_increase_call(
+                      "https://ecomm-demo.utpalpati.repl.co/cart",
+                      {
+                        items: { itemId: id, qty: item.qty + 1 }
+                      }
+                    );
+                  }}
+                >
+                  +
+                </button>
+              </div>
+            </>
+          );
+        }
+        return returnobj;
+      },
+      <button
+        class="cart"
+        style={{
+          cursor: inStock ? "pointer" : "not-allowed",
+          pointerEvents: inStock ? "auto" : "none"
+        }}
+        onClick={() => {
+          cart_add_call("https://ecomm-demo.utpalpati.repl.co/cart", {
+            items: { itemId: id, qty: 1 }
+          });
+        }}
+      >
+        {inStock ? "Add to Bag" : "Out of Stock"}
+      </button>
+    );
+  }
+
+  async function wishlist_add_call(url, payload) {
+    try {
+      let response = await axios.post(url, payload);
+      if (response.status === 200) {
+        dispatch({
+          type: "ADD_WISH",
+          payload: payload.itemId
+        });
+      }
+    } catch (err) {
+      console.log(err);
+    }
+  }
+  async function wishlist_remove_call(url, payload) {
+    try {
+      let response = await axios.delete(url, { data: payload });
+      if (response.status === 200) {
+        dispatch({
+          type: "REMOVE_WISH",
+          payload: payload.itemId
+        });
+      }
+    } catch (err) {
+      console.log(err);
+    }
+  }
+  function Add_to_wishlist_button(id) {
+    return wishlist.reduce(
+      (returnobj, item) => {
+        if (item === id) {
+          return (
+            <>
+              <button
+                class="wish"
+                onClick={() => {
+                  wishlist_remove_call(
+                    "https://ecomm-demo.utpalpati.repl.co/wishlist",
+                    { itemId: id }
+                  );
+                }}
+              >
+                <i class="fas fa-heart" style={{ color: "rgb(186, 0, 0)" }}></i>
+              </button>
+            </>
+          );
+        }
+        return returnobj;
+      },
+      <button
+        class="wish"
+        onClick={() => {
+          wishlist_add_call("https://ecomm-demo.utpalpati.repl.co/wishlist", {
+            itemId: id
+          });
+        }}
+      >
+        <i class="far fa-heart"></i>
+      </button>
+    );
+  }
+
+  return (
+    <>
+      <div className="ecom-navbar">
+        <Link to="/">
+          <div className="logo">
+            <img
+              src="http://www.pngmart.com/files/7/Growing-Plant-PNG-Pic.png"
+              alt="logo"
+            />
```

Comment:
The logo image is not visible.

Reply:
Thanks for pointing this out. In firefox it was working but in other browsers it is not working. I will fix it.